packages/devtools_app/lib/src/screens/memory/panes/diff/diff_pane.dart

```diff
@@ -15,9 +23,270 @@ class DiffPane extends StatefulWidget {
   State<DiffPane> createState() => _DiffPaneState();
 }
 
+/// Will notify if count of items or current index changed.
+class _DiffPaneController with ChangeNotifier {
```

Comment:
what is the benefit of mixing in ChangeNotifier instead of just having the controller class have ValueNotifier members? I'm not seeing anywhere where we call `diffPaneController.listen`, so what value is this adding?

Reply:
We can just use a `ListValueNotifier` for `snapshots` and then we are already using a `ValueNotifier` for `index` that other parts of the code can listen to. This will reduce an added layer of notifier complexity, and would be consistent with the rest of the DevTools codebase as a whole.